hudi-metaserver/src/test/java/org/apache/hudi/metaserver/store/TestRelationDBBasedStore.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.metaserver.store;
+
+import org.apache.hudi.metaserver.HoodieMetaServer;
+import org.apache.hudi.metaserver.thrift.NoSuchObjectException;
+import org.apache.hudi.metaserver.thrift.TAction;
+import org.apache.hudi.metaserver.thrift.THoodieInstant;
+import org.apache.hudi.metaserver.thrift.TState;
+import org.apache.hudi.metaserver.thrift.MetaStoreException;
+import org.apache.hudi.metaserver.thrift.Table;
+import org.apache.thrift.TException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Unit tests on metadata store base on relation database of hoodie meta server.
+ */
+public class TestRelationDBBasedStore {
+
+  private MetadataStore store;
+  private final String db = "test_db";
+  private final String tb = "test_tb";
+
+  @Test
+  public void testAPIs() throws TException {
+    HoodieMetaServer.getEmbeddedMetaServer();
+    store = HoodieMetaServer.getMetadataStore();
+    testTableRelatedAPIs();
+    testTimelineRelatedAPIs();
+  }
+
+  private void testTableRelatedAPIs() throws MetaStoreException, NoSuchObjectException {
+    Assertions.assertTrue(store.createDatabase(db));
```

Comment:
let's avoid the style of `Assertions.assertTrue`. it increases code verbosity. we should use static import instead.

Reply:
will fix